### DISCUSSION TITLE
Previous table with a trigger migrated with native

### DISCUSSION THREAD

**Discussion by jdesouza** [pgpartman/pg_partman#543] [General]
I am migrating an existing table with partman. This table has a trigger when a data is inserted, one column gets updated based on the input. Wondering if the trigger is propagated to the partitions from default and how is the best way to refer the partition names. I tried to create the trigger with the parent name but got some error.

**Comment by keithf4** [comment]
Honestly haven't done a lot of work with triggers and how they're inherited with native partitioning yet myself. If you can provide the full schema for the partition set and the trigger code I can try to look into it when I have some time.

**Reply by jdesouza** [reply]
Thanks
